Cargo.toml

```diff
@@ -192,7 +192,7 @@ ignored = ["napi", "oxc_transform_napi", "prettyplease"]
 [profile.dev]
 # Disabling debug info speeds up local and CI builds,
 # and we don't rely on it for debugging that much.
-debug = false
+# debug = false
```

Comment:
Please undo this

Reply:
```suggestion
```

Reply:
unintentional. Reverted. Note- this is an issue for running a debugger in intellij.

Reply:
Yup, same here for me in VSCode. I simply don't commit it; it helps keep CI nice and speedy 🏃